app/components/developers/banner_component.rb

```diff
@@ -0,0 +1,20 @@
+module Developers
+  class BannerComponent < ApplicationComponent
+    private attr_reader :banners, :selected_banner
```

Comment:
```suggestion
    attr_reader :selected_banner
    private attr_reader :banners
```

It doesn't make a difference for View Components but it shows better intent.